scripts/process-mints.js

```diff
@@ -43,6 +32,27 @@ async function main() {
     );
 
     console.log('resultFromLocalIpfsNode', resultFromLocalIpfsNode);
+  }
+
+  const queue = fastq.promise(uploadResource, config.MINTS_QUEUE_COCURRENCY);
+
+  const oldEvents = await Ethernauts.queryFilter('Transfer');
+
+  const proccesed = oldEvents
```

Comment:
So, an image is considered uploaded if its corresponding mint event has been received?

I can think of several scenarios where this bot will fail:
What if the upload failed?
What if the mint event occurred while this bot was down?

IMO the algorithm should be:
* Get past mint events
* Start listening to incoming new mint events
* Past two steps keep a dynamic queue
* A cyclic verifier checks which of those mint events don't have their corresponding asset uploaded by actually checking for the asset on IPFS
* To save resources, assets already checked can be marked in an internal data structure
* Assets identified as needing to be uploaded are sent into another queue for upload

Reply:
So, taking into account the feedback, I think it would be best to have 2 separate scripts. One that's constantly running listening for **new** mint events, and doesn't have to keep track of the history and starts super fast (because it doesn't have to check for all the assets), and also could be scaled to run in parallel. And the other one, which should be ran on a cronjob (to be sure that we didn't left nothing behind), which will only check for past mint events and be sure that everything was uploaded correctly.

Reply:
Feedback taken, I'm going to merge this PR for now and address the fixes on other PRs